less/home-page.less

```diff
@@ -72,4 +145,42 @@
       }
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+// Contact Us styling
+
+form {
+  margin: 0 auto;
+  width: 500px;
+  padding: 1em;
+  border: 1px solid black;
+  border-radius: 1em;
+}
+
+form div {
+  margin-top: 1em;
+}
+  
+label {
+  display: inline-block;
+  width: 90px;
+  text-align: right;
+}
+
+input, textarea {
+  font: 1em sans-serif;
+  width: 100px;
+  box-sizing: border-box;
+  border: 1px solid black;
+}
+
+input: focus, textarea: focus {
+  border-color: #000;
+}
+
+textarea {
+  vertical-align: top;
```

Comment:
try to utlize flexbox instead to align things. Mix and matching is possible, but not really recommended.